uptycs/manifest.json

```diff
@@ -0,0 +1,60 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "d27ee4b6-649d-42bd-b7ac-fb40537d7031",
+  "app_id": "uptycs",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect alerts, detections data from Uptycs platform",
+    "title": "Uptycs",
+    "media": [
+     {
+      "media_type": "image",
+      "caption": "Uptycs events dashboard",
+      "image_url": "images/integration_dashboard_1.png"
+     },
+     {
+      "media_type": "image",
+      "caption": "Uptycs events per host trend graph",
```

Comment:
Can you please re-crop this image or take a new screenshot so that it is not cut off? Note that all images need to have an aspect ratio of 16:9 and follow our [media guidelines](https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#media-carousel).

Reply:
updated the screenshot as per the guidelines.

Reply:
Hi @cgadde-uptycs, although the images meet our size guidelines, they are cut off. Can you resize the browser window so that it fits the 16:9 aspect ratio but the dashboard doesnt get cut off? This applies to your `integration_dashboard_2` image as well.

Reply:
Thanks @anawishnoff updated both images.